tests/test_data/text2codeExample-cz.xml

```diff
@@ -1,31 +0,0 @@
-<nodes>
```

Comment:
Please leave this file here (or you can create `yet_unused_test_data` folder or some similar one..